cmd/service.go

```diff
@@ -26,6 +26,7 @@ func setupServiceCommand() *cobraext.Command {
 		RunE:  upCommandAction,
 	}
 	upCommand.Flags().StringP(cobraext.DataStreamFlagName, "d", "", cobraext.DataStreamFlagDescription)
+	upCommand.Flags().StringP(cobraext.VariantFlagName, "s", "", cobraext.VariantFlagDescription)
```

Comment:
Nit. Why `s`? I think I would prefer to don't have an abbreviated form :slightly_smiling_face:

Reply:
It's for `service variant`, it's because `-v` is reserved for `verbose`. I can drop the short form in this case :)

Reply:
Fixed